pypsa/optimization/constraints.py

```diff
@@ -233,10 +233,16 @@ def define_operational_constraints_for_committables(n, sns, c):
         n.model.add_constraints(status, "=", 0, name, mask=mask)
 
     # linearized approximation because committable can partly start up and shut down
-    # only valid additional constraints if start up costs equal to shut down costs
     cost_equal = all(
         n.df(c).loc[com_i, "start_up_cost"] == n.df(c).loc[com_i, "shut_down_cost"]
     )
+    # only valid additional constraints if start up costs equal to shut down costs
+    if n._linearized_uc and not cost_equal:
+        logger.warning("The linear relaxation of the unit commitment cannot be "
+                       "tightened since the start up costs are not equal to the "
+                      "shut down costs. Proceed with the linear relaxation "
+                      "without the tightening by additional constraints. "
+                      "This might result in a longer solving time.")
```

Comment:
Good question. I am not if the solution is getting less correct compared to the linearized UC with the additional constraint but I am quite sure that since the problem is less tight it probably leads to longer solving times. That's why I have added the statement to the warning.

 We corresponded with Bowen Hua who is the lead author of the [paper]( https://ieeexplore.ieee.org/document/8000610) from where we got the additional constraints. He said that the proof of these additional constraints relies on the start up costs being equal to the shut down costs and that it would not be trivial to modify the constraints for unequal start up and shut down costs. He also suggested that one could run the relaxed unit commitment without the additional constraints.

Reply:
> He also suggested that one could run the relaxed unit commitment without the additional constraints.

I guess that is the most important